src/systems/filecoin_blockchain/struct/block_producer/_index.md

```diff
@@ -42,22 +42,31 @@ Anytime a miner receives new blocks, it should evaluate what is the heaviest Tip
 
 ## Block Creation
 
-Scratching a winning ticket, and armed with a valid `ElectionProof`, a miner can now publish a new block!
+Scratching (a) winning ElectionPoSt ticket(s), and armed with the requisite `ElectionPoStOutput`, a miner can now publish a new block!
 
 To create a block, the eligible miner must compute a few fields:
 
-- `Ticket` - new ticket generated from that in the prior epoch (see {{<sref ticket_generation>}}).
-- `ElectionProof` - A specific signature over the min_ticket from `randomness_lokkback` epochs back (see {{<sref leader_election>}}).
-- `ParentWeight` - The parent chain's weight (see {{<sref chain_selection>}}).
 - `Parents` - the CIDs of the parent blocks.
-- `ParentState` - Note that it will not end up in the newly generated block, but is necessary to compute to generate other fields. To compute this:
+- `ParentWeight` - The parent chain's weight (see {{<sref chain_selection>}}).
+- `ParentState` - This is the state of the chain after all of the message from each of the `Parents` have been applied to their own `ParentState`. To compute this:
   - Take the `ParentState` of one of the blocks in the chosen parent set (invariant: this is the same value for all blocks in a given parent set).
   - For each block in the parent set, ordered by their tickets:
     - Apply each message in the block to the parent state, in order. If a message was already applied in a previous block, skip it.
     - Transaction fees are given to the miner of the block that the first occurance of the message is included in. If there are two blocks in the parent set, and they both contain the exact same set of messages, the second one will receive no fees.
     - It is valid for messages in two different blocks of the parent set to conflict, that is, A conflicting message from the combined set of messages will always error.  Regardless of conflicts all messages are applied to the state.
```

Comment:
Remove most of this text and link instead to the interpreter's description of tipset execution.